actionview/lib/action_view/renderer/partial_renderer/collection_caching.rb

```diff
@@ -0,0 +1,74 @@
+require 'active_support/core_ext/object/try'
+
+module ActionView
+  module CollectionCaching
+    extend ActiveSupport::Concern
+
+    included do
+      # Use Rails.cache or the fallback cache store if Action View is used without Rails.
+      mattr_accessor(:collection_cache) do
+        Rails.try(:cache) || ActiveSupport::Cache::MemoryStore.new
```

Comment:
You're right. I had it as a railtie first, but then I switched to this. I should have just checked the console:
```bash
kaspth:action view kasperhansen$ irb -raction_view
irb(main):001:0> Rails
NameError: uninitialized constant Rails
```